analyses/efo-mondo-mapping/README.md

```diff
@@ -0,0 +1,22 @@
+# EFO and MONDO mapping
+
+
+## Introduction and Purpose
+
+The Experimental Factor Ontology (EFO) is a resource that provides systematic description of experimental variables including cancer groups available in [European Bioinformatics Institute](https://www.ebi.ac.uk/) databases and some other external projects. 
+The scope of EFO is to support the annotation, analysis and visualization of data handled by many groups at the EBI and as the core ontology for [OpenTargets](https://www.opentargets.org/ "OpenTargets' Homepage").
```

Comment:
```suggestion
The scope of EFO is to support the annotation, analysis and visualization of data handled by many groups at the EBI and as the core ontology for [OpenTargets](https://www.opentargets.org/).
```
This doesn't seem to have gone through - I just updated what it should look like, though. Here is a section on [links](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links).

Reply:
@jharenza That's weird. I pushed this change last week. It looks fine in my branch. 
Anyway, thank you.